pages/_app.tsx

```diff
@@ -142,7 +143,7 @@ function MyApp({ Component, pageProps }: ComponentWithPageLayout) {
           provider: localStorageProvider,
         }}
       >
-        {/* <Toaster position="top-center" /> */}
+        <NextNProgress color="#f97316" height={4} />
```

Comment:
Where is the orange color coming from? I searched the project for `#f97316` and it's no where to be found.

Do we want to used the OpenSauced orange, `hsla(19, 100%, 50%, 1)`? 

https://github.com/brainconnect93/open-sauced-app/blob/d2e73a9c29d06c949cc5a81ca82b6d29fbf755cd/tailwind.config.js#L36

Reply:
> Do we want to used the OpenSauced orange, hsla(19, 100%, 50%, 1)?

Yes!... will update to the hsl version👍